### ISSUE TITLE
Question about multi_lr_scheduler?

### ISSUE THREAD

**Issue by Xiaolong-RRL** [PaddlePaddle/Paddle3D#337]
想问一下，如何在paddle3d的config文件中设置不同的学习率策略？要求是前5个epoch采用cosine形式的warmup，之后采取steplr形式，在90和120个epoch分别变为原来的十分之一，一共训练140个epoch

**Comment by Xiaolong-RRL** [comment]
目前可行的方法：
```python
import math
from paddle.optimizer.lr import LRScheduler

class CosineWarmupMultiStepDecayByEpoch(LRScheduler):
    def __init__(self, 
                 learning_rate,
                 warmup_steps, 
                 start_lr, 
                 milestones, 
                 decay_rate, 
                 end_lr = None):
        
        self.warmup_epochs = warmup_steps
        self.start_lr = start_lr
        self.end_lr = end_lr if end_lr is not None else learning_rate
        self.milestones = milestones
        self.decay_rate = decay_rate
        super(CosineWarmupMultiStepDecayByEpoch, self).__init__(learning_rate)
        
    def get_lr(self):
        epoch = self.last_epoch + 1
        # warmup
        if self.last_epoch < self.warmup_epochs:
            # cosine
            return self.start_lr + (self.end_lr - self.start_lr) * (1 - math.cos(math.pi * self.last_epoch / self.warmup_epochs)) / 2
        else:
            if epoch in self.milestones:
                self.end_lr *= self.decay_rate
                return self.end_lr
            else:
                return self.end_lr
